web/src/main/java/org/springframework/security/web/ObservationFilterChainDecorator.java

```diff
@@ -568,13 +569,12 @@ public String getContextualName(FilterChainObservationContext context) {
 
 		@Override
 		public KeyValues getLowCardinalityKeyValues(FilterChainObservationContext context) {
-			KeyValues kv = KeyValues.of(CHAIN_SIZE_NAME, String.valueOf(context.getChainSize()))
+			return KeyValues.of(CHAIN_SIZE_NAME, String.valueOf(context.getChainSize()))
```

Comment:
Will you please make the same change in `ObservationWebFIlterChainDecorator`, the reactive version?

Reply:
Done